### ISSUE TITLE
According to Web Content Accessibility Guidelines (WCAG) - every iframe must have a title attribute

### ISSUE THREAD

**Issue by BogdanCerovac** [googlemaps/js-samples#216]
My first bug report, I hope I will clarify my points :)

When doing some accessibility reviews I've noticed that title attribute for Google Maps iframe embeds is missing.

1. Checked documentation - unable to find some description about importance of title (added a comment on the site about it) - developers and end-users should understand the value the title is providing for assistive technologies.
2. Checked multiple examples - it looks like none of them is using and/or mentioning title (I may missed some, but in general I think it is missing),
3. Tested online version of Google maps -> share -> Embed and I think there we should have some smart defaults for the title (mentioned below) and the users could maybe edit the title with some simple form (required title input for example).

#### Environment details

1. Have not checked all examples but it looks like all <iframe> elements are missing title attribute.
2. Tested online Google maps embed UX and it is missing the title attribute

#### Steps to reproduce

  1. open example and check iframe in the DOM tree / or use a accessibility testing tool (like aXe or similar) / use a screenreader

#### Code example

When generating / showing iframe with Google maps we should provide a title attribute for it (it should be meaningful of course - for example "Showing {name of place / coordinates } on Google Maps" or something that gives it a good description;
```
<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d4027.4752040795925!2d9.580445431436969!3d59.68729112714207!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4640c5aca7b86cdf%3A0x826e7df571bc58fb!2sLinnertdammen!5e0!3m2!1sen!2sno!4v1596010580601!5m2!1sen!2sno" width="300" height="300" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0" title="Showing map of Linnertdammen (Kongsberg, Norway, Europe) on Google Maps"></iframe>
```

Links with references;
- https://www.w3.org/TR/WCAG20-TECHS/H64.html
- https://webaim.org/techniques/frames/
- https://dequeuniversity.com/tips/provide-iframe-titles

Thanks!
